jujugui/static/gui/src/app/components/inspector/test-inspector.js

```diff
@@ -44,4 +49,25 @@ describe('Inspector', function() {
           <div>child</div>
         </div>);
   });
+
+  it('calls the changeState callable when on header click', function() {
```

Comment:
It's not a callback because it's just a function call that's being called from the click callback.